### PR TITLE
fixed git error messages

### PR DESCRIPTION
**Pull request by Luschgy**

### CODE REVIEW

File: jenkins/prepareOskar.ps1 [hunk 1]
```diff
@@ -1,4 +1,19 @@
-﻿$HDD = $(Split-Path -Qualifier $env:WORKSPACE)
+﻿Function proc($process,$argument)
+{
+    $p = Start-Process $process -ArgumentList $argument -NoNewWindow -PassThru
+    $h = $p.Handle
```
Comment (KVS85) [inline jenkins/prepareOskar.ps1]: Why do we need this handle?

File: jenkins/prepareOskar.ps1 [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+    $p.WaitForExit()
```
Comment (KVS85) [inline jenkins/prepareOskar.ps1]: Can we use `-Wait` parameter in `Start-Process` instead of this?

Comment (Luschgy) [reply jenkins/prepareOskar.ps1]: No because the -wait parameter is broken in Powershell

File: jenkins/prepareOskar.ps1 (hunk 1 above)
Comment (Luschgy) [reply jenkins/prepareOskar.ps1]: To fix the broken Powershell -wait parameter. Don't ask me why it works, it does. Also it is just a copy of the Funktion in the main helper.ps1

**Pull request merged**